services/app-api/handlers/forms/post/generateQuarterForms.js

```diff
@@ -22,6 +23,8 @@ export const main = handler(async (event, context) => {
     return null;
   }
 
+  await authorizeAdmin(event);
```

Comment:
This will break the quarterly job without the other changes _but_ that also should be a bit out there until it needs to trigger.

We either need the alternate handler or agree we're following up with the other change quickly.

Reply:
I think an alternate handler is the ideal solution. It feels odd to me to have a single lambda that handles two different types of event.